WORKSPACE

```diff
@@ -234,3 +191,36 @@ scala_maven_import_external(
         "https://repo.maven.apache.org/maven2/",
     ],
 )
+
+RULES_JVM_EXTERNAL_TAG = "2.9"
+RULES_JVM_EXTERNAL_SHA = "e5b97a31a3e8feed91636f42e19b11c49487b85e5de2f387c999ea14d77c7f45"
+
+http_archive(
+    name = "rules_jvm_external",
+    strip_prefix = "rules_jvm_external-%s" % RULES_JVM_EXTERNAL_TAG,
+    sha256 = RULES_JVM_EXTERNAL_SHA,
+    url = "https://github.com/bazelbuild/rules_jvm_external/archive/%s.zip" % RULES_JVM_EXTERNAL_TAG,
+)
+
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+
+maven_install(
+    name = "maven",
+    artifacts = [
+        # test of strict deps (scalac plugin UT + E2E)
+        "com.google.guava:guava:21.0",
+        # test adding a scala jar:
+        "com.twitter:scalding-date_2.11:0.17.0",
```

Comment:
you've removed the scala_mvn_artifact code here, which will complicate/break users of 2.12 (or 2.13). Can you use the `scala_mvn_artifact` code as the original code did to not regress?

Reply:
done.